src/services/goToDefinition.ts

```diff
@@ -12,10 +12,14 @@ namespace ts.GoToDefinition {
         if (node === sourceFile) {
             return undefined;
         }
-        const { parent } = node;
 
+        const { parent } = node;
         const typeChecker = program.getTypeChecker();
 
+        if (node.kind === SyntaxKind.OverrideKeyword || node.kind === SyntaxKind.JSDocOverrideTag) {

```

Comment:
I guess `JSDocOverrideTag` contains:

```ts
/**
* @override and comments
*/
foo (){}
```

You might only want to check `tagName` for `JSDocTag`.

Reply:
Do you mean that the entire JSDoc tag should be clickable?

Reply:
Nope. Sorry I haven't try the current behavior but **seems** It will trigger at all `JSDocOverrideTag` which may includes the comments.

Would you mind add a test to cover:

```ts
/**
* @override and /*trigger here*/comments 
* */ 
foo (){}
```

Reply:
I've added test [goToDefinitionOverriddenMember14.ts](https://github.com/microsoft/TypeScript/blob/2c6e1acca09cdae0e5430c40b3027588a87a2b68/tests/cases/fourslash/goToDefinitionOverriddenMember14.ts).

Reply:
Does the [goToDefinitionOverriddenMember11](https://github.com/microsoft/TypeScript/blob/fd4e9e428679d70ada8c759dfc9a148c1112afe9/tests/cases/fourslash/goToDefinitionOverriddenMember11.ts) cover expected behavior?